### PR TITLE
Previous optimisation in findStringLen has broken unescaping of \\\" sequences

### PR DESCRIPTION
**Pull request by kirillx**

### CODE REVIEW

**Comment by kirillx** [comment]
@GoWebProd plz pay attention

File: jlexer/lexer.go [hunk 1]
```diff
@@ -253,6 +253,16 @@ func findStringLen(data []byte) (isValid bool, length int) {
 		if idx == 0 || (idx > 0 && data[idx-1] != '\\') {
 			return true, length + idx
 		}
+
+		// count \\\\\\\ sequences. even number of slashes means quote is not really escaped
+		cnt := 1
```
Comment (GoWebProd) [inline jlexer/lexer.go:258]: maybe set `cnt := 0` and remove `cnt-1` in code below?

Comment (kirillx) [reply jlexer/lexer.go:258]: up to you, i just prefer cnt to mean real "count" rather than smth else.

**Comment by GoWebProd** [comment]
Thanks

**Pull request merged**